CONTRIBUTING.md

```diff
@@ -77,6 +77,14 @@ Nobody is perfect, and sometimes we mess things up. That said, here are some goo
 
 ## Pull Request Q&A
 
+### How do I find a good first issue to work on?
+
+Check the GitHub issues page for issues labeled "good first issue". You can also:
+- Look for typos in comments or documentation
```

Comment:
I'm sure it would be an easy first issue, I'm not sure if it qualifies as "good".